fragments/k8s/cdk-aws/README.md

```diff
@@ -0,0 +1,534 @@
+# The Canonical Distribution of Kubernetes - designed for Amazon Web Services (AWS) Public Cloud
```

Comment:
I think we do not want to repeat the full README here, we would rather add a section on the readme with the "facilities" the integration charm offers.